recipe/meta.yaml

```
@@ -54,6 +54,8 @@ requirements:
     - cyrus-sasl =2.1.27
     - msgpack-c
     - aws-sdk-cpp
+    - azure-identity-cpp
```

We also need to explicitly add `azure-core-cpp` for now.
```suggestion
    - azure-core-cpp
    - azure-identity-cpp
```